### DISCUSSION TITLE
not detecting birds anymore

### DISCUSSION THREAD

**Discussion by Klages** [mcguirepr89/BirdNET-Pi#364] [Q&A] [answered]
Somehow, my BirdNET-Pi is not detecting birds anymore. I checked if all services are running and also rebooted my device, when I listen to the microphone I can hear everything, so I rule out that there is an issue with my microphone.  I tried to play birdsongs with my phone, which is clearly visible in the Spectrogram, but nothing is detected. Does anyone have a clue?
![firefox_JcEWQ8vwmz](https://user-images.githubusercontent.com/3393617/173130291-525bfc9c-2341-4d90-b201-4063414505af.png)

**Comment by nilspupils** [comment]
Did you check the system info for running services? Also, you can have a look at the log and check for errors. Post here what you find.

**Comment by Klages** [comment]
Services are running:
![image](https://user-images.githubusercontent.com/3393617/173141457-860bd764-356c-4f19-bf1c-e5c6b74d58c4.png)

Log output when i play bird songs from my mobile:
![image](https://user-images.githubusercontent.com/3393617/173141495-c71ac70c-4452-4c83-930f-7323b9c3471d.png)

**Reply by CaiusX** [reply]
Hi. 

I had a similar problem this week, didn’t get to the bottom of it because system started again after reboot.

Possibly save your logs before rebooting/restarting services to try get to the bottom of this.

**Comment by Klages** [comment, marked as answer]
I reinstalled BirdNET on a new Brand micro SD, I was using a cheap Chinese micro SD before. 
Overall, a lot of things seem to run faster now, I will report if the issue comes up again.
